types/account.go

```diff
@@ -120,14 +79,13 @@ func (acc EthAccount) MarshalYAML() (interface{}, error) {
 func (acc EthAccount) MarshalJSON() ([]byte, error) {
 	var ethAddress = ""
 
-	if acc.BaseAccount != nil && acc.Address != nil {
+	if acc.BaseAccount != nil && acc.Address == "" {
```

Comment:
not sure if this `==` a typo, but seems that condition here has been the opposit way.